examples/Grill-Tag-107_2-conversion.txt

```diff
@@ -3,7 +3,6 @@
 0 _ * r 0._
 0.1 1 * r 0.11
 0.1 _ * r 0.1_
-0.11 1 * r 0.111
```

Comment:
oops, didn't mean to commit this, relates to #3 which needs to be investigated properly.
Was just doing the minimum to display the graph output without errors...